pkg/storegateway/gateway_test.go

```diff
@@ -397,11 +391,6 @@ func TestStoreGateway_BlocksSyncWithDefaultSharding_RingTopologyChangedAfterScal
 		createBucketIndex(t, bucketClient, userID)
 	}
 
-	// Randomise the seed but log it in case we need to reproduce the test on failure.
```

Comment:
Same comment about custom rng and logging the seed as above.

Reply:
@pstibrany the test doesn't use `rand` though. Are you thinking of possible implicit uses of `rand` in code that the test calls?